migrations/20190911155158_car-dealer.js

```diff
@@ -0,0 +1,20 @@
+
+exports.up = function(knex) {
+  return knex.schema.createTable('car-dealer', tbl => {
+      tbl.increments()
+      tbl.text('make')
+        .notNullable(); 
+      tbl.text('model')
+        .notNullable(); 
+      tbl.text('VIN')
+        .unique()
+        .notNullable(); 
+      tbl.integer('mileage')
```

Comment:
try to make use of the second input variable for int, string, text, and other sql table creators(the char limit)
so you could have done integer('milage', 20) 
this way someone cant input a super bizarre number that would be impossible in real world terms